packages/emotion/src/__tests__/useTheme.test.tsx

```diff
@@ -185,112 +118,11 @@ describe('useTheme hook', () => {
       expect(callback).to.have.been.calledWith(canvas)
     })
   })
-  describe('with InstUISettingsProvider and global theme', () => {
```

Comment:
please do not delete this, here we are not testing for something that can be done by Chromatic, but the business logic of the theming engine